webrtc.html

```diff
@@ -16540,6 +16540,7 @@ <h4>
             When a {{MediaStream}} is created to represent a stream obtained
             from a remote peer, the {{MediaStream/id}} attribute is initialized
             from information provided by the remote source.
+            The {{MediaStream}}'s [[\CanFireEvent]] slot is set to <code>true</code>.
```

Comment:
As discussed at the editors meeting, 
streams are held alive by
receiver.[[[AssociatedRemoteMediaStreams]]](https://w3c.github.io/webrtc-pc/#dfn-associatedremotemediastreams), so this shouldn't be needed. 

We could perhaps clear that internal slot in close() to allow stream (and track) GC in that edge case.

Reply:
I updated the PR to clear the remote media streams slots at transceiver stop time.